packages/core/src/App/Containers/trade-notifications.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { observer } from 'mobx-react';
+import { MobileWrapper, Money, SwipeableNotification, Text } from '@deriv/components';
+import { useStore } from '@deriv/stores';
+import { getCardLabels, getContractPath } from '@deriv/shared';
+import { localize, Localize } from '@deriv/translations';
+
+const TradeNotifications = observer(({ show_trade_notifications }: { show_trade_notifications?: boolean }) => {
+    const {
+        notifications: { removeTradeNotifications, trade_notifications },
+    } = useStore();
+    if (!show_trade_notifications || !trade_notifications.length) return null;
+
+    return (
+        <MobileWrapper>
+            <div className='trade-notifications'>
+                {trade_notifications.slice(0, 3).map(notification => {
+                    const { buy_price, contract_id, currency, contract_type, id, profit, symbol, status, timestamp } =
+                        notification;
+                    return (
+                        <SwipeableNotification
+                            key={id}
+                            is_failure={status !== 'open' && profit < 0}
```

Comment:
Nitpick: I'd say it's better to call it `is_loss`. Failure is not a proper opposite of success in financial terms

Reply:
@akmal-deriv I made SwipeableNotification to be a reusable component which can be used for any content, and is_loss is specifically related to DTrader contracts. is_failure  and is_success where selected intentionally and only used to force a particular background-color )